.php_cs

```diff
@@ -21,40 +21,80 @@ $config->setRules(
         'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'binary_operator_spaces' => true,
+        'blank_line_after_opening_tag' => true,
         'blank_line_before_statement' => [
-            'statements' => ['return', 'continue', 'case', 'default', 'declare', 'throw', 'try'],
+            'statements' => [
+                'break',
+                'case',
+                'continue',
+                'declare',
+                'default',
+                'die',
+                'do',
+                'exit',
+                'for',
+                'foreach',
+                'goto',
+                'if',
+                'include',
+                'include_once',
+                'require',
+                'require_once',
+                'return',
+                'switch',
+                'throw',
+                'try',
+                'while',
+                'yield',
+            ],
         ],
         'cast_spaces' => ['space' => 'none'],
-        'class_attributes_separation' => ['elements' => ['method']],
+        'class_attributes_separation' => ['elements' => ['method', 'property']],
+        'combine_consecutive_issets' => true,
+        'combine_consecutive_unsets' => true,
         'compact_nullable_typehint' => true,
         'concat_space' => ['spacing' => 'one'],
+        'date_time_immutable' => true,
         'declare_equal_normalize' => true,
+        'declare_strict_types' => true,
+        'dir_constant' => true,
         'explicit_indirect_variable' => true,
         'explicit_string_variable' => true,
+        'final_internal_class' => true,
+        'fopen_flag_order' => true,
+        'fopen_flags' => true,
         'fully_qualified_strict_types' => true,
+        'function_to_constant' => true,
         'function_typehint_space' => true,
+        'heredoc_indentation' => true,
         'heredoc_to_nowdoc' => true,
+        'implode_call' => true,
         'include' => true,
         'linebreak_after_opening_tag' => true,
         'list_syntax' => ['syntax' => 'short'],
+        'logical_operators' => true,
```

Comment:
I'm all for this but if applied on a legacy codebase it could change the desired behaviour.

Reply:
This is "sane defaults" - not for everything.  This is intended to be a starting point, and can aleays be changed.